docs/accessibility/high-contrast.md

```diff
@@ -0,0 +1,7 @@
+# High Contrast
+
+The high contrast helps people to locate and distinguish between the different visual elements in the MakeCode editor. This is enabled by each MakeCode target with its own use of color and contrast. So, a high contrast view is not always available in every instance of a MakeCode editor (partner editions).
+
+The high contrast mode in the editor is available even when the operating system configuration hasn't enabled it. In MakeCode, it is enabled manually from either the ``Hidden Tab Menu`` or the ``Settings`` menu.
+
+![](/static/images/accessibility-highcontrast.png)
```

Comment:
Add a developer section that explains shortly how to enable high contrast

Reply:
Once again, not convinced by the fact to put technical documentation in an end users documentation. There is a specific section in the summary for developers. End users don't have to see technical points.